felix/bpf-gpl/conntrack_types.h

```diff
@@ -53,6 +53,8 @@ struct calico_ct_leg {
 
 	__u32 opener:1;
 
+	__u32 workload:1;
```

Comment:
Can imagine forgetting that this means "the source of this leg is a local workload iface" at some point. Worth a comment just to make ti really clear?